content/en/tracing/profiling/_index.md

```diff
@@ -129,6 +129,59 @@ The Datadog Profiler requires Python 2.7+. Memory profiling only works on Python
 [3]: /tracing/visualization/#services
 {{% /tab %}}
 
+{{% tab "Go" %}}
+
+The Datadog Profiler requires Go 1.12+. To begin profiling applications:
+
+1. Install `ddtrace` which contains both tracing and profiling:
```

Comment:
This sounds weird in terms of `go`. `ddtrace` is a package, `tracer` is another package, `profiler` is yet again another package. They aren't necessarily contained in one another. We should say something like:
```suggestion
1. Get `dd-trace-go` using the command:
```

Reply:
+1